core/inject.js

```diff
@@ -59,7 +59,8 @@ const inject = function(container, opt_options) {
   }
   const options =
       new Options(opt_options || (/** @type {!BlocklyOptions} */ ({})));
-  const subContainer = document.createElement('div');
+  const subContainer =
+      /** @type {!HTMLDivElement} */ (document.createElement('div'));
```

Comment:
If only it would work to type the variable rather than the initialiser:
```
  const /** !HTMLDivElement */ subContainer = document.createElement('div');
```
Alas…